src/pages/guide/casino.mdx

```diff
@@ -536,6 +546,32 @@ The code will be: 3472
 
 </Dropdown>
 
+<Header title="Chosen" id="chosen">
+	1 players
+</Header>
+
+After assembling the ATM code, you can insert the credit card. You will then be prompted to enter a PIN code, but be careful—you only have three attempts. 
+
+Once the correct PIN is entered, wait a few seconds for the money to be dispensed. Afterward, collect the money and place it in the coin exchanger to receive two yellow coins and one green coin.
+
+One person must collect the green coin and use it on a gambling machine. After a short animation showing a win, the player will receive a lucky message. Following this, interact with the fates machine; unlike before, rather than perishing, you will receive an electric shock and be granted the Chosen Title. Go outside and approach the arcade machine to shock and activate it.
```

Comment:
Pas besoin de s
```suggestion
One person must collect the green coin and use it on a gambling machine. After a short animation showing a win, the player will receive a lucky message. Following this, interact with the fate machine; unlike before, rather than perishing, you will receive an electric shock and be granted the Chosen Title. Go outside and approach the arcade machine to shock and activate it.
```